src/astro/sql/operators/agnostic_save_file.py

```diff
@@ -55,7 +51,8 @@ def execute(self, context: dict) -> None:
         """
```

Comment:
Github didn't allow me to comment on lines 33-36 :see_no_evil: So here it goes:

The properties:
*  `output_file_path`
* `output_conn_id`
* `output_file_format`

Can be wrapped into a single argument `output_file: File`